### PR TITLE
Feature/Initial setups for Strings and Colors

### PR DESCRIPTION
**Pull request by Aslanyan99n**
- Add Localization
- Add app icon
- Add Separate groups

### CODE REVIEW

**Review by Mekhak11 — APPROVED** [review]

**Review by iskandaryansona — APPROVED** [review]

File: Birthday/Birthday/Utils/Shape/RoundedCornersShape.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//  RoundedCornersShape.swift
+//  Birthday
+//
+//  Created by Narek on 21.10.24.
+//
+
+import SwiftUI
+
+struct RoundedCornersShape: Shape {
+    let corners: UIRectCorner
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Utils/Shape/RoundedCornersShape.swift]: Add spaces before first property of object and closing braces too

Comment (nellyAslanyan) [reply Birthday/Birthday/Utils/Shape/RoundedCornersShape.swift]: And I'm not sure UIKit integration is necessary here, SwiftUI is providing enough modifiers.

File: Birthday/Birthday/Utils/Localization/Localize.swift [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    return String(format: string.localized, arguments: arguments)
+}
+
+func localizedPlural(_ string: String, argument: CVarArg) -> String {
+    return string.localizedPlural(argument)
+}
+
+extension String {
+    var localized: String {
+        return localized(using: nil, in: .main)
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Utils/Localization/Localize.swift]: try to omit extra returns

File: Birthday/Birthday/Utils/Localization/Localize.swift [hunk 3]
```diff
@@ -0,0 +1,127 @@
+//
+//  Localize.swift
+//  Birthday
+//
+//  Created by Narek on 21.10.24.
+//
+
+import Foundation
+/// Internal current language key
+let LCLCurrentLanguageKey = "LCLCurrentLanguageKey"
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Utils/Localization/Localize.swift]: I do believe there is more simple way to make string localized :)

File: Birthday/Birthday/Utils/Extensions/Texts.swift [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+//  Texts.swift
+//  Birthday
+//
+//  Created by Narek on 21.10.24.
+//
+
+import Foundation
+
+extension String {
+    enum Button {
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Utils/Extensions/Texts.swift]: here too :D

File: Birthday/Birthday/Utils/Shape/RoundedCornersShape.swift (hunk 1 above)
Comment (Aslanyan99n) [reply Birthday/Birthday/Utils/Shape/RoundedCornersShape.swift]: Removed

File: Birthday/Birthday/Utils/Localization/Localize.swift (hunk 2 above)
Comment (Aslanyan99n) [reply Birthday/Birthday/Utils/Localization/Localize.swift]: Changed

File: Birthday/Birthday/Utils/Localization/Localize.swift (hunk 3 above)
Comment (Aslanyan99n) [reply Birthday/Birthday/Utils/Localization/Localize.swift]: Changed

**Review by nellyAslanyan — APPROVED** [review]

**Pull request merged**